back/engines/free/email_campaigns/app/models/email_campaigns/campaigns/invite_received.rb

```diff
@@ -49,7 +49,12 @@ def mailer_class
     end
 
     def activity_triggers
-      { 'Invite' => { 'created' => true } }
+      {
+        'Invite' => {
+          'created' => true,
+          'resent' => true
```

Comment:
Is this definitely an OR? Sure you've tested it. Just wanted to be sure.

Reply:
Yeah it worked with mailcatcher locally. But I will confirm on staging too before releasing :+1: